libs/testserver/fake_workspace.go

```diff
@@ -33,40 +32,32 @@ func NewFakeWorkspace() *FakeWorkspace {
 	}
 }
 
-func (s *FakeWorkspace) WorkspaceGetStatus(path string) (workspace.ObjectInfo, int) {
+func (s *FakeWorkspace) WorkspaceGetStatus(path string) *workspace.ObjectInfo {
 	if s.directories[path] {
-		return workspace.ObjectInfo{
+		return &workspace.ObjectInfo{
 			ObjectType: "DIRECTORY",
 			Path:       path,
-		}, http.StatusOK
+		}
 	} else if _, ok := s.files[path]; ok {
-		return workspace.ObjectInfo{
+		return &workspace.ObjectInfo{
 			ObjectType: "FILE",
 			Path:       path,
 			Language:   "SCALA",
-		}, http.StatusOK
+		}
 	} else {
-		return workspace.ObjectInfo{}, http.StatusNotFound
+		return nil
```

Comment:
It's a bit too implicit that `nil` corresponds to 404. Normally, in Go, returning nil does not correspond to an error. 

Should we explicitly return a 404 here?

Reply:
Fair enough. I thought it would be generally applicable, but this is the only case that relies on it. Updated to return explicit response here.